convert.go

```diff
@@ -59,7 +60,7 @@ func Move(oldPath string, newPath string, out io.Writer) error {
 	}
 	newDS.deactivate()
 
```

Comment:
Move() should probably take a context eventually. Fine for now.

Reply:
yeah I didn't bother since it's only used by the CLI